src/adios4dolfinx/legacy_readers.py

```diff
@@ -9,13 +9,13 @@
 
 from mpi4py import MPI
 
-import adios2
 import basix
 import dolfinx
 import numpy as np
 import numpy.typing as npt
 import ufl
 
+from .adios2_helpers import adios2
```

Comment:
Would you prefer each of these relative imports to be changed to the try/catch import of adios2?

Reply:
Hmm, probably, or use the definition i suggested above.
`from .adios_utils import import_adios2; adios2 = import_adios2()`